time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -119,14 +142,18 @@ def find_all(
 
         custom_params = self.generate_params(date_range)
 
-        return CosmosDBRepository.find_all(
+        time_entries = CosmosDBRepository.find_all(
```

Comment:
is this returning the time-entries belonging to the JWT user or all of them?
Can you please restrict to only the time-entries belonging to the user in the JWT?

Reply:
>is this returning the time-entries belonging to the JWT user or all of them?

Yes. That is how it works.

Check out the user_id/owner_id value is passed in the dictionary conditions, in the dao:
```
class TimeEntriesCosmosDBDao(APICosmosDBDao, TimeEntriesDao):
...
   def get_all(self, conditions: dict = {}) -> list:
        event_ctx = self.create_event_context("read-many")
        conditions.update({"owner_id": event_ctx.user_id})

        date_range = self.handle_date_filter_args(args=conditions)
        return self.repository.find_all(
            event_ctx, conditions=conditions, date_range=date_range
        )
```